bears/vcs/git/GitCommitBear.py

```diff
@@ -204,3 +224,81 @@ def check_body(self, body,
             yield Result(self, 'Body of HEAD commit contains too long lines. '
                                'Commit body lines should not exceed {} '
                                'characters.'.format(body_line_length))
+
+    def check_issue(self, body,
+                    detect_host: bool=False,
+                    require_url: str=None,
+                    keyword_regexp: str=None):
+        """
+        Check for matching issue related URLs.
+
+        :param body:                Body of the commit message of HEAD.
+        :param detect_host:         Whether to automatically detect host
+                                    from git config or not.
+        :param require_url:         Whether the commit message requires
```

Comment:
As there are only two values, this should be a `bool`.
Default disabled, as urls are not mandatory on github/gitlab.  This means `.coafile` in coala repos need to enable this flag.